odoo/tools/query.py

```diff
@@ -57,6 +57,7 @@ def __init__(self, cr, alias, table=None):
 
         # joins {alias: (kind, table, condition, condition_params)}
         self._joins = {}
+        self._raw_joins = {}  # {query: params}
```

Comment:
Why not a list of 2-items tuples maybe ? Do we need the specific key-based behavior ?